thermal_processing.py

```diff
@@ -151,7 +109,7 @@ def update_metadata(recording, api):
         # metadata["device_name"] = reader.device_name
 
         if reader.preview_secs:
-            metadata["additionalMetadata"] = {"previewSecs": reader.preview_secs}
+            metadata["additionalMetadata"] = {"Preview secs": reader.preview_secs}
```

Comment:
I don't mind this but I thought it would be better to fix this at presentation time (in the frontend)